### PR TITLE
Minor fixes

### LINKED ISSUE #506: Redundant processing variables and systems
A mistake i've made in past coding, i've added in many boolean variables in various places to record whether an object is currently being processed, not realising that one already exists, is_processing

These redundant systems need to be removed and the existing system used.

Farthermore in at least one case (the ripper) I wrote an entirely redundant timer-chaining processing system when SSfastprocess would have done the job.

And lastly, this wasnt just my mistake, a few existing items in baycode also have this problem

Items which need to be fixed:

Baycode:
Biogenerator (code\game\machinery\biogenerator.dm)
Computer (code\game\machinery\computer\computer.dm)
Effect System: (code\game\objects\effects\effect_system.dm)
Beehive: (code\modules\hydroponics\beekeeping\beehive.dm)
Mint: (code\modules\mining\mint.dm)
Spell: (baystation12\code\modules\spells\spell_code.dm)

DS13:
Corruption Nodes: (code\modules\necromorph\corruption_nodes.dm)
Heavy Pulse Rifle: (code\modules\projectiles\guns\ds13\heavy_pulse.dm)
Ripper: (code\modules\projectiles\guns\special\ripper\sawblade.dm)

### PR DESCRIPTION
**Pull request by DTraitor**
### Purpose
Fixes: https://github.com/DS-13-Dev-Team/DS13/issues/506;
Fixed recharger deconstruction;
Fixed R&D Console not showing "Print x5" and "Print x10" buttons;
Changed Seeker price to 15000;
Changed incend divet ammo protolathe cost. Now require some oxygen and solid phoron;
Added force gun to tech tree and protolathe;
ERT channel is now ":q" instead of nothing.

### CODE REVIEW

File: code/game/objects/effects/effect_system.dm [hunk 1]
```diff
@@ -363,7 +363,7 @@ steam.start() -- spawns the effect
 /////////////////////////////////////////////
 /datum/effect/effect/system/trail
 	var/turf/oldposition
-	var/processing = 1
+	is_processing = 1
```
Comment (NanakoAC) [inline code/game/objects/effects/effect_system.dm]: never directly set this

First of all, its set by the START/STOP_PROCESSING macros.

Secondly, it doesn't take integers, iirc it's a reference to a processing subsystem

File: code/modules/mining/mint.dm [hunk 2]
```diff
@@ -124,7 +123,7 @@
 	if(href_list["makeCoins"])
 		var/temp_coins = coinsToProduce
 		if (src.output)
-			processing = 1;
+			is_processing = 1;
```
Comment (NanakoAC) [inline code/modules/mining/mint.dm]: as previously mentioned, no direct setting of this.

File: code/modules/projectiles/guns/special/ripper/ripper.dm [hunk 3]
```diff
@@ -27,7 +27,7 @@
 
 
 
-	has_safety = FALSE	//Safety switches are for military/police weapons, not for tools
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/special/ripper/ripper.dm:30]: thats not how safety works. Power tools don't typically have a switch that prevents them from working, they have a dead man's switch that makes it only operate while the trigger is held

The concept of a thing that prevents firing is a thing for firearms, not for tools. Please revert this change

**Review by NanakoAC — CHANGES_REQUESTED** [review]
lots of incorrect usage of is_processing, and the safety thing

File: code/modules/projectiles/guns/special/ripper/ripper.dm (hunk 3 above)
Comment (ghost) [reply code/modules/projectiles/guns/special/ripper/ripper.dm:30]: @NanakoAC Citing real life, most power tools that involve drills, large saws and blades do have safeties. I'd be shocked if something of the Ripper's caliber didn't have something to prevent it from misfiring.

File: code/game/objects/effects/effect_system.dm (hunk 1 above)
Comment (DTraitor) [reply code/game/objects/effects/effect_system.dm]: This is what you asked in https://github.com/DS-13-Dev-Team/DS13/issues/506. Or did I understood it wrong?

File: code/modules/projectiles/guns/special/ripper/ripper.dm (hunk 3 above)
Comment (DTraitor) [reply code/modules/projectiles/guns/special/ripper/ripper.dm:30]: Drills, large saws, blades? I have electrical screwdriver that has safety.

File: code/modules/hydroponics/beekeeping/beehive.dm [hunk 4]
```diff
@@ -170,11 +170,11 @@
 	anchored = 1
 	density = 1
 
-	var/processing = 0
+	var/curently_processing = 0	//Used to track what is inside, not to check if it's processing
```
Comment (NanakoAC) [inline code/modules/hydroponics/beekeeping/beehive.dm]: make the name of this not contain "process" at all to avoid ambiguity

Comment (NanakoAC) [reply code/modules/hydroponics/beekeeping/beehive.dm]: maybe just "working"

**Review by NanakoAC — CHANGES_REQUESTED** [review]
just one little thing with beehive

**Review by NanakoAC — APPROVED** [review]
